include/aa_permitted_pipelines_generic.hpp

```diff
@@ -809,14 +809,14 @@ namespace astroaccelerate {
 
 			timer.Stop();
 			time_log.adding("Periodicity", "total", timer.Elapsed());
-
-			float time = timer.Elapsed()/1000;
-			printf("\n\n === OVERALL PERIODICITY THROUGHPUT INCLUDING SYNCS AND DATA TRANSFERS ===\n");
-
-			printf("\nPerformed Periodicity Location: %f (GPU estimate)", time);
-			printf("\nAmount of telescope time processed: %f", tstart_local);
-			printf("\nNumber of samples processed: %ld", inc);
-			printf("\nReal-time speedup factor: %f\n", (tstart_local) / (time));
+			time_log.adding("Total", "total", timer.Elapsed());
+
+			//float time = timer.Elapsed()/1000;
+			// printf("\n\n === OVERALL PERIODICITY THROUGHPUT INCLUDING SYNCS AND DATA TRANSFERS ===\n");
+			// printf("\nPerformed Periodicity Location: %f (GPU estimate)", time);
+			// printf("\nAmount of telescope time processed: %f", tstart_local);
+			// printf("\nNumber of samples processed: %ld", inc);
+			// printf("\nReal-time speedup factor: %f\n", (tstart_local) / (time) );
```

Comment:
This is in the case of periodicity. Does the old autotune script looking for the value of Real-time speed-up?

Reply:
No autotune does not optimize periodicity. Besides these values of real-time speed-up are confusing since they are only for the periodicity search without any other module.